Sources/Fluent/Query/Join.swift

```diff
@@ -45,6 +45,10 @@ public final class Pivot<
         }
     }
 
+    public var idKey: String {
+        return Pivot.database!.driver.idKey
```

Comment:
You should use one of the entities to get the `idKey`. This will crash if the pivot isn't added as a migration.

You should get the idKey like this:
```swift
let idKey = try First.query().idKey
```